ts-embed_assets/index.ts

```diff
@@ -1,8 +1,8 @@
-import express = require('express')
+import * as express from 'express'
 import * as hello from './hello'
 import * as content from './static'
 
-export const app = express()
+const app = express()
```

Comment:
did this need to change?

Reply:
yeah we dont support esModuleInterop flag which causes this to see the app listener as exports.app instead of app in the dispatcher